packages/manager/modules/enterprise-cloud-database/src/service/get-started/get-started.controller.js

```diff
@@ -0,0 +1,147 @@
+import find from 'lodash/find';
+import get from 'lodash/get';
+import isUndefined from 'lodash/isUndefined';
+import set from 'lodash/set';
+
+export default class EnterpriseCloudDatabaseServiceGetStartedCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    CucCloudMessage,
+    CucControllerHelper,
+    enterpriseCloudDatabaseService,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.CucControllerHelper = CucControllerHelper;
+    this.enterpriseCloudDatabaseService = enterpriseCloudDatabaseService;
+  }
+
+  $onInit() {
+    this.stepperIndex = 0;
+    this.data = {
+      clusterName: this.clusterDetails.name,
+      dailyBackup: false,
+      replicaConfig: {
+        replicaCount: 0,
+        useDefaultPaymentMethod: true,
+      },
+    };
+    this.loaders = {
+      savingSecuritySettings: false,
+      savingSettings: false,
+    };
+
+    this.subscribeToMessages();
```

Comment:
```suggestion
    this.loadMessages();
```
to keep consistency with everywhere else

Reply:
This is done